common/src/main/java/com/revenuecat/purchases/common/ReceiptInfo.kt

```diff
@@ -3,7 +3,7 @@ package com.revenuecat.purchases.common
 import com.revenuecat.purchases.models.PricingPhase
 import com.revenuecat.purchases.models.StoreProduct
 
-class ReceiptInfo(
```

Comment:
this used to be a data class until SkuDetails was added: https://github.com/RevenueCat/purchases-android/pull/167/files